packages/@expo/cli/ts-declarations/@react-native-dev-middleware/index.d.ts

```diff
@@ -1,176 +1,5 @@
-    _app: string;
-    /** Stores socket connection between Inspector Proxy and device. */
-    _deviceSocket: WS;
-    /** Stores last list of device's pages. */
-    _pages: Page[];
-    /** Stores information about currently connected debugger (if any). */
-    _debuggerConnection: DebuggerInfo | null = null;
-    /** Whether we are in the middle of a reload in the REACT_NATIVE_RELOADABLE_PAGE. */
-    _isReloading: boolean = false;
-    /** The previous "GetPages" message, for deduplication in debug logs. */
-    _lastGetPagesMessage: string = '';
-    /** Mapping built from scriptParsed events and used to fetch file content in `Debugger.getScriptSource`. */
-    _scriptIdToSourcePathMapping: Map<string, string>;
-    /** Root of the project used for relative to absolute source path conversion. */
-    _projectRoot: string;
-
-    /**
-     * Last known Page ID of the React Native page.
-     * This is used by debugger connections that don't have PageID specified
-     * (and will interact with the latest React Native page).
-     */
-    _lastConnectedReactNativePage: Page | null = null;
-
-    constructor(id: number, name: string, app: string, socket: WS, projectRoot: string);
-
-    getName(): string;
-    getPagesList(): Page[];
-
-    /**
-     * Handles new debugger connection to this device:
-     * 1. Sends connect event to device
-     * 2. Forwards all messages from the debugger to device as wrappedEvent
-     * 3. Sends disconnect event to device when debugger connection socket closes.
-     */
-    handleDebuggerConnection(socket: WS, pageId: string): void;
-
-    /**
-     * Handles messages received from device:
-     * 1. For getPages responses updates local _pages list.
-     * 2. All other messages are forwarded to debugger as wrappedEvent.
-     *
-     * In the future more logic will be added to this method for modifying
-     * some of the messages (like updating messages with source maps and file locations).
-     */
-    _handleMessageFromDevice(message: MessageFromDevice): void;
-
-    /** Sends single message to device. */
-    _sendMessageToDevice(message: MessageToDevice): void;
-
-    /** Sends 'getPages' request to device every PAGES_POLLING_INTERVAL milliseconds.*/
-    _setPagesPolling(): void;
-
-    /** Allows to make changes in incoming message from device. */
-    _processMessageFromDevice(
-      payload: { method: string; params: { sourceMapURL: string; url: string } },
-      debuggerInfo: DebuggerInfo
-    ): void;
-
-    /** Allows to make changes in incoming messages from debugger. */
-    _interceptMessageFromDebugger(
-      request: DebuggerRequest,
-      debuggerInfo: DebuggerInfo,
-      socket: WS
-    ): DebuggerResponse | null;
-
-    // _newReactNativePage
-    // _processDebuggerSetBreakpointByUrl
-    // _processDebuggerGetScriptSource
-    // _mapToDevicePageId
-  }
+declare module '@react-native/dev-middleware' {
+  type DebuggerInfo = NonNullable<unstable_Device['_debuggerConnection']>;
```

Comment:
Note to self, check if this is still necessary, or if we can use it in a better way (more `@react-native/dev-middleware`-contract compliant way)